webpack.config.ts

```diff
@@ -18,6 +18,9 @@ import GeneratePackageJsonPlugin from 'generate-package-json-webpack-plugin';
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
 import ThreadsPlugin from 'threads-plugin';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import nodeExternals from 'webpack-node-externals';
```

Comment:
install this: https://www.npmjs.com/package/@types/webpack-node-externals

then u dont need to use ts-ignore :)

Reply:
Is it worth downloading a package to remove 2 lines of ignore comment? :D

Reply:
ive added type definitions for the other webpack pages though :P

Reply:
What other webpack pages?